build-aux/snap/snapcraft.yaml

```
@@ -195,7 +195,7 @@ parts:
     plugin: nil
     source: .
     build-snaps:
-      - go/1.18/stable # the default Go toolchain
+      - go/latest/stable # the default Go toolchain
```

Should we not be more intentional about changing versions?
Preferably we only need to update a single line to update to the same go version for both snaps and debs, however this could be a next step.

---

I'm slightly torn about this one. On the one hand, we should likely be doing this already, on the other this will build with whatever happens to be in the store at a given moment. I know the Go team has a good track record, but it's not like the were no bugs in the releases.

---

What if we set this to go/1.23/stable?

---

That feels like a decent trade-off. Does syntax allow to pin specific revision?

---

It's the same as before, we simply follow the latest 1.23 release
```suggestion
      - go/1.23/stable # the default Go toolchain
```

---

@bboozzoo , Samuele preference is to be consistent with version used for fips

---

we had terrible changes of memery usage behavior with some go version switches, I wouldn't use latest that way, it also means we have no control what is used at various points of a release, as the go version might change under our feet